src/components/Budget/BudgetForm/BudgetForm.js

```diff
@@ -85,37 +85,42 @@ const BudgetForm = ({
               md={8}
               mdOffset={2}
             >
-              <Row end="xs">
-                <Col xs={12}>
-                  <ExpandAllButton
-                    accordionStatus={sections}
-                    onToggle={expandAll}
-                  />
-                </Col>
-              </Row>
-              <AccordionSet
-                accordionStatus={sections}
-                onToggle={toggleSection}
-              >
-                <Accordion
-                  label={<FormattedMessage id="ui-finance.budget.summary.title" />}
-                  id={SECTIONS_BUDGET.SUMMARY}
-                >
-                  {initialValues.metadata && <ViewMetaData metadata={initialValues.metadata} />}
-                  <BudgetSummary
-                    budget={initialValues}
-                  />
-                </Accordion>
-                <Accordion
-                  label={<FormattedMessage id="ui-finance.budget.information.title" />}
-                  id={SECTIONS_BUDGET.INFORMATION}
-                >
-                  <BudgetInformationFields
-                    fiscalEnd={periodEnd}
-                    fiscalStart={periodStart}
-                  />
-                </Accordion>
-              </AccordionSet>
+              <AccordionStatus>
+                <Row end="xs">
+                  <Col xs={12}>
+                    <ExpandAllButton />
+                  </Col>
+                </Row>
+                <AccordionSet>
+                  <Accordion
+                    label={<FormattedMessage id="ui-finance.budget.summary.title" />}
+                    id={SECTIONS_BUDGET.SUMMARY}
+                  >
+                    {initialValues.metadata && <ViewMetaData metadata={initialValues.metadata} />}
```

Comment:
FYI, we were doing like this before because `ViewMetaData` was failing with exception in case of absent `metadata`. Now it works fine, and hiding/showing blocks is not the best UX. It's better than exception, but now it should be fine